style.css

```diff
@@ -663,6 +663,14 @@ a.src-btn {
   padding-bottom: 20px;
 }
 
+li.tools-li-src {
+  background-color: #ebebff;
+  padding: 5px 10px;
+  width: 52px;
+  border-radius: 8px;
+  padding-left: 11px;
+}
+
 img.nature-animated {
   display: none;
```

Comment:
- [ ] Please make your popup close to the figma design, some don't match the Figma design yet: 
- Kindly blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- Remove second popup displayed on both mobile and desktop version
-  It would be better if the background is not scrollable when the popup page is open. you can use CSS property `overflow: hidden;` to prevent this.

- [ ] According to the project requiement, The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).
-  Please add more padding-top between the list info and the popup image as provided in Figma
-  Please add more padding-botton after the modal description and `See live`, `See source` button

Your Implementation | Figma Design
:---:|:---:
![today2](https://user-images.githubusercontent.com/79058364/163398982-c0e606fe-4718-4f29-87ee-efd23d0a72a7.png)|![today3](https://user-images.githubusercontent.com/79058364/163398933-541800fb-decb-4be8-ba54-2b19f247f3c0.png)

-  Adding these features will make your design look great and look exactly like the Figma design.